atf2tei.py

```diff
@@ -159,16 +153,18 @@ def normalize_transliteration(words):
         word = re.sub(r'H,', 'Ḫ', word)     # \u1E2A
         word = re.sub(r'j', 'ŋ', word)      # \u014B
         word = re.sub(r'J', 'Ŋ', word)      # \u014A
-        '''XML-escape the result.'''
+        'Convert damage marks to half-brackets.'
```

Comment:
Can we use this as well: https://tei-c.org/release/doc/tei-p5-doc/en/html/ref-damageSpan.html

Reply:
I see, thanks. Based on offline discussions this morning, I'll go ahead and merge this since it looks slightly better. I need to do the collection and markup thing for logogrammes anyway, so will make both changes at the same time, probably through a new class that does xml serialization of atf line markup.

I opened #6 to track that.